### PR TITLE
Add slides for PR creation and approval flow

### LINKED ISSUE #54: Improve lab setup slides
Add information on tagging logic
Link approval checklist https://github.com/anz-bank/go-course#pre-pr-checklist

### Explain labelling for own PR:
We use labels to avoid concurrent reviews and filter more easily on PRs that require action.
When your PR is ready:
- label with lab number e.g. `lab1`
- label with `Ready for review` (first time and after changes)
- wait for review and labels `Changes requested`or `Approved by colleague`
- wait for code CODEOWNER (trainer) to approve and add tag `Merge me`
- merge PR 

###  Explain labelling for reviewing others' PRs:
Review somebody else's PR in order to get own PR merged.     
Add link to review comments of other PR to own PR descriptions (does not have to be the same lab).
- add label `Under review` while reviewing 
- add label `Changes request` or `Approved by colleague` when review is done

### PR DESCRIPTION
**Pull request by anzboi**
Add two slides explaining how PR's get reviewed in the course.
One for participents role as a programmer, and the other as a
reviewer.

Also rearrange the slides regarding creating a PR to flow more
logically

Deployed slides: https://pr58-dot-gotraining-testing.appspot.com
Relevant slides: 45-47, 49-50

Fixes #54

### CODE REVIEW

File: content/gocourse-2019-05-melbourne.slide [hunk 1]
```diff
@@ -586,7 +578,7 @@ If `master` has advanced after `feature` branch and PR creation you have two opt
 
 	language:none
 	git checkout master
-	git pull upstream master # or just `git pull` if `master` is set to track `upstream/master`
+	git pull upstream master # or just `git` `pull` if `master` is set to track `upstream/master`
```
Comment (juliaogris) [inline content/gocourse-2019-05-melbourne.slide]: Revert please - this is already a code block (indented) so the back ticks are rendered ([go present syntax](https://godoc.org/golang.org/x/tools/present)).

File: content/gocourse-2019-05-melbourne.slide [hunk 2]
```diff
@@ -530,21 +530,11 @@ Document them
...
-* Repository setup overview
-
-Clone your fork of [[https://github.com/anz-bank/go-course][anz-bank/go-course]]
-
-Create PRs
-
--  from `feature` branch of your own fork (remote `origin`)
--  to `master` on [[https://github.com/anz-bank/go-course][anz-bank/go-course]] (remote `upstream`)
-
 * Repository setup details
```
Comment (juliaogris) [inline content/gocourse-2019-05-melbourne.slide]: Maybe just `Repository setup` setup seeing as you deleted the previous slide?

File: content/gocourse-2019-05-melbourne.slide [hunk 3]
```diff
@@ -602,7 +594,18 @@ Before adding any commits to `feature` locally run:
...
+- Tag with the appropriate lab tag (eg: `lab1`)
+- When ready, tag with `Ready` `for` `review`
+- Wait for PR to be tagged `Approved` `by` `colleague` or `Changed` `requested`
+- If approved by colleague, wait for a CODEOWNER to approve and tag with `Merge` `me`
+- Merge
+
+* Lab PR approval flow - Reviewer
 
+- Remove `ready for review` and add `under review`
+- Tag with either `Approved by colleague` or `changes requested`
```
Comment (juliaogris) [inline content/gocourse-2019-05-melbourne.slide]: and remove `under review` ?

Comment (juliaogris) [reply content/gocourse-2019-05-melbourne.slide]: alsoe: probably backtick around every single word.

**Review by juliaogris — COMMENTED** [review]
Thanks for adding this.

Apologies in advance for being so OCD, but this is also what I think we should require from the students so it seems like I really should insist here:

PR description and all commit messages in "imperative mood" please, like a command, e.g.: `Add slides....`.

Also, would be super awesome if you could deploy the slides for review, see initial slack comments on #gc-melbs-instructors.

I've added you with your ANZ account to the `anzedu` gcp project.

If you write `Fixes #54` rather than `[54]` in your PR description, issue `#54` will automatically get closed when PR is merged.

File: content/gocourse-2019-05-melbourne.slide [hunk 4]
```diff
@@ -602,7 +594,18 @@ Before adding any commits to `feature` locally run:
 	language:bash
 	git pull origin feature
 
+* Lab PR approval flow - Programmer
+
+- Tag with the appropriate lab tag (eg: `lab1`)
+- When ready, tag with `Ready` `for` `review`
+- Wait for PR to be tagged `Approved` `by` `colleague` or `Changed` `requested`
```
Comment (camscale) [inline content/gocourse-2019-05-melbourne.slide]: s/Changed/Changes/

File: content/gocourse-2019-05-melbourne.slide [hunk 5]
```diff
@@ -602,7 +594,18 @@ Before adding any commits to `feature` locally run:
...
+- If approved by colleague, wait for a CODEOWNER to approve and tag with `Merge` `me`
+- Merge
+
+* Lab PR approval flow - Reviewer
 
+- Remove `ready for review` and add `under review`
```
Comment (camscale) [inline content/gocourse-2019-05-melbourne.slide]: \`Ready\` \`for\` \`review\` and add \`Under\` \`review\`

(capitalisation and spacing)

File: content/gocourse-2019-05-melbourne.slide (hunk 3 above)
Comment (camscale) [inline content/gocourse-2019-05-melbourne.slide]: ditto

**Comment by juliaogris** [comment]
I've also added @camh-anz as reviewer cause he knows the slides better than anybody else who is a native English speaker ;)

File: content/gocourse-2019-05-melbourne.slide (hunk 3 above)
Comment (anzboi) [reply content/gocourse-2019-05-melbourne.slide]: Argh, I went back and added a bunch of these after realising why they are needed, must have missed these ones, thanks 👍 .

**Comment by anzboi** [comment]
I'll do my best to adopt @juliaogris suggestions from here on, bad habits.

**Review by juliaogris — COMMENTED** [review]
LGTM, @camh-anz ?

**Comment by camscale** [comment]
Is this published anywhere? pr58-blah.whatever.the.url.is? I'd like to review the output.

**Comment by anzboi** [comment]
Deployed slides: https://pr58-dot-gotraining-testing.appspot.com
Relevant slides: 45-47, 49-50

**Review by camscale — APPROVED** [review]
LGTM

**Pull request merged**